ask-sdk-webservice-support/ask_sdk_webservice_support/verifier.py

```diff
@@ -244,7 +244,7 @@ def _validate_certificate_url(self, cert_url):
                 "Signature Certificate URL has invalid hostname: {}. "
                 "Expecting {}".format(hostname, CERT_CHAIN_URL_HOSTNAME))
 
-        normalized_path = os.path.normpath(parsed_url.path)
+        normalized_path = (os.path.normpath(parsed_url.path)).replace('\\', '/')
```

Comment:
Since [normpath](https://docs.python.org/2/library/os.path.html#os.path.normpath) converts the path to window's referenced path, this seems to be needed. Thanks for the PR 👍 . 

However, I think it is better off to rather change the [`CERT_CHAIN_URL_STARTPATH`](https://github.com/alexa/alexa-skills-kit-sdk-for-python/blob/master/ask-sdk-webservice-support/ask_sdk_webservice_support/verifier_constants.py#L39) constant to OS specific value, considering the path separator might be a different one in some other OS. Can you revert this change and instead make the change in the other file, something similar to using [`os.path.sep`](https://docs.python.org/2/library/os.html#os.sep) and the constant string value `echo.api`?

Also, it would be great if you can add a unit test for this, something similar to [this test](https://github.com/alexa/alexa-skills-kit-sdk-for-python/blob/master/ask-sdk-webservice-support/tests/unit/test_verifier.py#L155)?